pkgs/development/compilers/swift/default.nix

```diff
@@ -15,7 +16,9 @@ let
     callPackage = newScope self;
 
     # Current versions of Swift on Darwin require macOS SDK 10.15 at least.
-    # Re-export this so we can rely on the minimum Swift SDK elsewhere.
+    # The Swift compiler propagates the 10.15 SDK and deployment target.
+    # Packages that need a newer version can add it to their build inputs
+    # to use it (as normal).
     apple_sdk = pkgs.darwin.apple_sdk_11_0;
```

Comment:
Comment is inconsistent with the code (I guess the conditional got dropped at some point?).